### PR TITLE
adding in files

### PR DESCRIPTION
**Pull request by rnburn**

### CODE REVIEW

File: go/client/main.go [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+	// Set up a connection to the server.
+	conn, err := grpc.Dial(address, grpc.WithInsecure())
+	if err != nil {
+		log.Fatalf("did not connect: %v", err)
+	}
+	defer conn.Close()
+	c := pb.NewStoreClient(conn)
+
+	// Contact the server and print out its response.
+  item_id := int32(22)
```
Comment (frenchfrywpepper) [inline go/client/main.go]: indentation issue, have you run gofmt on the code?

File: py/tracer_interceptor.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+class TracerInterceptor(object):
+    def __init__(self, tracer):
+        self.tracer = tracer
+
+    def __call__(self, request, metadata, invoker):
+        print "before call to server"
```
Comment (frenchfrywpepper) [inline py/tracer_interceptor.py]: Remove print statements

**Review by frenchfrywpepper — CHANGES_REQUESTED** [review]
I'd like to see tox or setup.py, even a Makefile at root for both go and python. I'm having trouble getting the example servers running myself.

I'd also like to see (at a minimum) file level comments explaining the purpose of each file.

And I also think it'd be nice to separate out the example code from the core code that users of the package will want.

Unit tests would be great too.

**Comment by rnburn** [comment]
What errors are you getting when running the python client/server.

I'll add some comments to help navigate the files and reformat. But just to be clear this wasn't meant to be a package to install. It's just a staging area to develop the PR for this issue https://github.com/grpc/grpc/issues/8767
I expect to put the code into some files here https://github.com/grpc/grpc/tree/master/src/python/grpcio/grpc
and add tests somewhere in here
https://github.com/grpc/grpc/tree/master/src/python/grpcio_tests/tests

**Pull request closed without merging**